processing/src/main/java/io/druid/segment/IndexMerger.java

```diff
@@ -200,6 +200,9 @@ public File persist(
                 indexSpec.getBitmapSerdeFactory().getBitmapFactory()
             )
         ),
+        // if index is not rollup, then it should be not rollup here
+        // if index is rollup, then it is no need to rollup again
+        false,
```

Comment:
please modify the comment to reflect that true/false for persist.rollup, does not matter since there is only one iterable and we never really so any reOrdering while merging a single iterable.